templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,7 +13,9 @@
    "cell_type": "markdown",
```

Comment:
<p>We should delete this mention of moly_paths</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Should we also remove prompts for Moly points in <code>generate_tiled_region_fov_list</code>?</p>

Reply:
<p>No, we'll keep that in case advanced users want to use it, but the default will be that they'll say no</p>